### ISSUE TITLE
Time zone are not getting updated in ui

### ISSUE THREAD

**Issue by vpsidar** [nhn/tui.calendar#967]
## Version

1.15.1
## Development Environment

Windows, Chrome
## Current Behavior

Currently the ui shows the timezones which i have provided while initialization of calendar. I have changed the timezones via calender.setOptions() but the ui is not getting changed with respect to new zones.
![Screenshot (573)](https://user-images.githubusercontent.com/23169695/147624568-090c5d42-9013-400a-8fb8-b8f25badd2c5.png)
I am using the vue js and providing the timezones with props and updating it dynamically.

```js
// Calendar Create
createCalendar () {
      this.calendar = new TuiCalendar('#calendar', {
        defaultView: 'week', // set 'week' or 'day'
        timezone: {
          zones: this.setTimeZones(),
          offsetCalculator: function (timezoneName, timestamp) {
            // e.g. +09:00 => -540, -04:00 => 240
            return moment.tz.zone(timezoneName).utcOffset(timestamp)
          }
        },
        taskView: false,
        scheduleView: ['time']
      })
}

// updating the options  using props and watcher

watch: {
teacher_time_zone: {
      handler () {
        // this.schedules = []
        // this.render()
        // this.createCalendar()
        if (this.calendar) {
          this.calendar.setOptions({
            timezone: {
              zones: this.setTimeZones(),
              offsetCalculator: function (timezoneName, timestamp) {
                // e.g. +09:00 => -540, -04:00 => 240
                return moment.tz.zone(timezoneName).utcOffset(timestamp)
              }
            }
          }, true)
          this.calendar.render()
        }
      },
      deep: true,
      immediate: true
    }
}

```

## Expected Behavior

I want to have a feature where i can provide the timezone dynamically and the ui should be render accordingly.

**Comment by adhrinae** [comment]
@vpsidar 
Thanks for the report.
There's a missing point when the calendar updates options and applies them to views.

I'll try to release the fixed version next week.

**Comment by adhrinae** [comment]
@vpsidar 
I've published `v1.15.2` to resolve this issue. Could you check this out?
